docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -64,29 +64,44 @@ For generation of SGX evidence in ECDSA and EPID formats, the SGX quote-ex libra
-- The quote-ex library supports generation of SGX quote in multiple formats (including ECDSA-p256 and EPID variations). With quote-ex, quote generation is always done out-of-process by working with a background service (called AESM) on the local platform.
+#### Background: the SGX DCAP and quote-ex Libraries
 
-An SGX platform can have either the quote-ex library or the DCAP library, or both of them installed. When both libraries are installed, the quote-ex library takes precedence, as it supports more evidence formats.
+- The DCAP library only supports generation of SGX quotes in ECDSA-p256 format. With DCAP, the quote generation can be done either in-process, or out-of-process by working with a background service (called AESM) running on the same platform.
+- The quote-ex library supports generation of SGX quotes in multiple formats (including ECDSA-p256 and EPID variations). With quote-ex, quote generation is always done out-of-process by working with a background service (called AESM) on the local platform.
 
-The OE SDK host-side plugin library is changed to dynamically detect the presence of the two libraries, load one of them, and record the entry points of the loaded library for invocation in the flows described below.
+An SGX platform can have either the quote-ex library or the DCAP library, or both of them installed. Installation and configuration of the AESM background service is independent of the installation of the two libraries.
 
-#### Alternative: Only Use the quote-ex Library
+When both libraries and the background services are all installed and configured, the quote-ex should library takes precedence, as it supports more evidence formats.
 
-Run-time dynamic detection and loading of multiple shared libraries complicates implementation. It also increases risk of version mismatch between OE SDK and the dependent libraries that is not detected at build time via library headers.
+#### Options for Host-side Plugin Library Load of the SGX DCAP and quote-ex Libraries
 
-As described previously, the quote-ex library supports a superset of formats as compared to the DCAP library, though it always depends on a background service for quote generation. If on SGX platforms the OE SDK always installs with the AESM background service (as a hard dependency), then an alternative implementation is to use the libsgx-quote-ex library only (instead of the DCAP library). With this implementation, the libsgx-quote-ex library is linked to the OE SDK at build time.
+There are several options that the OE SDK host-side plugin library can choose to load and use the SGX DCAP and quote-ex libraries. Experiments in implementation will help make the most suitable decision. In any case, from the software stack point of view, the decision only impacts the implementation of the host-side plugin library, and not the enclave-side plugin library or the OCALL interface.
```

Comment:
Can't parse grammar of first sentence.   
(Also suggest adding line breaks in the markdown to split into multiple lines to make diffing easier... also applies to many other long lines in this markdown.)

Reply:
Let me rewrite the first sentence.

Sorry for the long lines. With "word wrap" turned-on in my vscode editor, I've not been paying much attention to line-breaking.

How about after addressing all the comments, I create yet another commit to break all the long lines (which will mess up diff of the whole document)?